cmake/FindHAL.cmake

```diff
@@ -359,6 +378,7 @@ foreach(COMP ${HAL_FIND_COMPONENTS_FAMILIES})
                                                     STM32::${FAMILY}${CORE_C} 
                                                     CMSIS::STM32::${FAMILY}${CORE_C})
         target_include_directories(HAL::STM32::${FAMILY}${CORE_C} INTERFACE "${HAL_${FAMILY}${CORE_U}_INCLUDE}")
+        target_compile_definitions(HAL::STM32::${FAMILY}${CORE_C} INTERFACE USE_HAL_DRIVER)
```

Comment:
What is `USE_HAL_DRIVER` ?

Reply:
Indeed, it's not needed if user includes `stm32${FAMILY_L}xx_hal.h`
It's only use is to include this file through file `Drivers\CMSIS\Device\ST\STM32MP1xx\Include\stm32${FAMILY_L}xx.h`

Reply:
Yes indeed, it seems to be a mistake. I remove it.